sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormTrainingClientTestBase.java

```diff
@@ -106,7 +105,8 @@ FormTrainingClientBuilder getFormTrainingClientBuilder(HttpClient httpClient,
         if (getTestMode() == TestMode.PLAYBACK) {
             builder.credential(new AzureKeyCredential(INVALID_KEY));
         } else {
-            builder.credential(new DefaultAzureCredentialBuilder().build());
+            builder.credential(new AzureKeyCredential(
```

Comment:
Is this updated to not use Default Credential because business cards endpoint is not working with AAD?

Reply:
Yes AAD is not working for Business Card now

Reply:
Now, as in it was working previously and stopped working?

Reply:
Talked offline. It works due to only problem from my desktop.